src/linear_algebra/matrix/basis.lean

```diff
@@ -140,17 +141,24 @@ open linear_map
 by { haveI := classical.dec_eq ι',
       rw [← basis_to_matrix_mul_linear_map_to_matrix b b', to_matrix_id, matrix.mul_one] }
 
+lemma sum_repr_aux (x : M) : ∑ (j : ι'), b.repr (b' j) i * b'.repr x j = b.repr x i :=
```

Comment:
I'm not very happy with this lemma: it's rather specific, but I can't find a good way to generalize it either. (And the name and location aren't optimal either.) I'm open to improvements!